lang/package.go

```diff
@@ -78,6 +84,12 @@ func NewPackageFromBuild(log logger.Logger, pkg *build.Package, opts ...PackageO
 
 	examples := doc.Examples(files...)
 
+	// Set this package variables is necessary for the [comment.DocLink] generation.
```

Comment:
Good to see the explanation what this seemingly simple assignment does during the run-time. Does this mean only a single package can be exported from the CLI? Does this change the original behaviour?

```suggestion
	// Setting this package variables is necessary for the [comment.DocLink] generation.
```

Reply:
No. You can still use the command for more than one package.